test/functional/ex_cmds/grep_spec.lua

```diff
@@ -7,16 +7,27 @@ describe(':grep', function()
   before_each(clear)
 
   it('does not hang on large input #2983', function()
-    if eval("executable('grep')") == 0 then
-      pending('missing "grep" command')
+    local whichgrep = eval('&grepprg')
+
+    if whichgrep == '' then
+      pending('no grepprg configured')
```

Comment:
?? this should never happen. tests always set up nvim with the same settings. or if not, then we should just fail.

Reply:
rebase and force-push is ok 👍

Reply:
> also, not to overcomplicate, but grep_spec.lua is indeed skipped on windows:

oh, that's a good to try `rg` in the test then. so if the test works on Windows now, please keep the `rg` support in it :)